<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>movieWindow</class>
 <widget class="QDialog" name="movieWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>556</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Neuer Film</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/resources.qrc">
    <normaloff>:/icons/logo.png</normaloff>:/icons/logo.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,1,0,1,0,0,0" columnstretch="0,0,0,0,0,0">
     <item row="1" column="1">
      <widget class="QSpinBox" name="medien">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Medien:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="laenge">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
       <property name="displayIntegerBase">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QDoubleSpinBox" name="preis">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string>€</string>
       </property>
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Länge:
(Minuten)</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Preis:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="5">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>15</number>
       </property>
       <item alignment="Qt::AlignRight">
        <widget class="QToolButton" name="schauspielerAdd">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="res/resources.qrc">
           <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="SchauspielerRemove">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="res/resources.qrc">
           <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="usk">
       <property name="styleSheet">
        <string notr="true">text-align:center</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>18</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>USK:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="6">
      <widget class="QTreeWidget" name="schauspielerTree">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="indentation">
        <number>5</number>
       </property>
       <property name="headerHidden">
        <bool>false</bool>
       </property>
       <column>
        <property name="text">
         <string>Schauspieler</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="1" colspan="5">
      <widget class="QLineEdit" name="titel">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="6">
      <widget class="QTreeWidget" name="kategorieTree">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>130</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="indentation">
        <number>5</number>
       </property>
       <column>
        <property name="text">
         <string>Kategorien</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="6" column="5">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignRight">
        <widget class="QToolButton" name="kategorieAdd">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="res/resources.qrc">
           <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="kategorieRemove">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="res/resources.qrc">
           <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Jahr:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QDateEdit" name="release">
       <property name="displayFormat">
        <string>yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Gekauft:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QDateEdit" name="gekauft">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QComboBox" name="typ"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Typ:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>titel</tabstop>
  <tabstop>medien</tabstop>
  <tabstop>laenge</tabstop>
  <tabstop>preis</tabstop>
  <tabstop>usk</tabstop>
  <tabstop>release</tabstop>
  <tabstop>gekauft</tabstop>
  <tabstop>typ</tabstop>
  <tabstop>kategorieAdd</tabstop>
  <tabstop>kategorieRemove</tabstop>
  <tabstop>schauspielerAdd</tabstop>
  <tabstop>SchauspielerRemove</tabstop>
  <tabstop>schauspielerTree</tabstop>
  <tabstop>kategorieTree</tabstop>
 </tabstops>
 <resources>
  <include location="res/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>movieWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>movieWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
